src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -359,7 +359,9 @@ public Position Position
         // If false then can only flick pages when the current animation/scroll as ended.
         private bool flickWhenAnimating = false;
         private PropertyNotification propertyNotification;
-        protected float finalTargetPosition;
+
+        // Let's consider more whether this needs to be set as protected.
+        private float finalTargetPosition;
```

Comment:
Is this change included to this commit?

Reply:
Just made the field not be opened as protected one.